tests/include/test/fake_external_rng_for_test.h

```diff
@@ -37,4 +37,21 @@ void mbedtls_test_enable_insecure_external_rng(void);
 void mbedtls_test_disable_insecure_external_rng(void);
 #endif /* MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG */
 
+#if defined(MBEDTLS_PLATFORM_GET_ENTROPY_ALT)
+
+#include <mbedtls/platform.h>
+
+/* Force return value or entropy content in mbedtls_platform_get_entropy()
+ * as follows:
+ * - if fail == 0 && forced_entropy_content == 0 then
```

Comment:
Forcing an entropy content of 0 will be an important test case in the future. Please use a different value to indicate full entropy. For example `SIZE_MAX`.